### PR TITLE
Remove snapshot from json plugin jsons

### PR DESCRIPTION
**Pull request by bdmogal**

### CODE REVIEW

File: cassandra-plugins/resources/cassandra-plugins-1.0.0-batch.json [hunk 1]
```diff
@@ -1,3 +1,3 @@
 {
-  "parents" : [ "system:cdap-etl-batch[3.2.0-SNAPSHOT,3.2.0-SNAPSHOT]" ]
+  "parents" : [ "system:cdap-etl-batch[3.2.0,3.2.0]" ]
```
Comment (albertshau) [inline cassandra-plugins/resources/cassandra-plugins-1.0.0-batch.json]: for a 3.2.x compatible branch, these version ranges should all be [3.2.0,3.3.0)

**Comment by wolf31o2** [comment]
:+1: LGTM so long as there's nothing hidden away somewhere

File: README.rst [hunk 2]
```diff
@@ -71,9 +71,9 @@ You can deploy plugins using CDAP CLI::
 
 Example for loading Cassandra Plugin (from cassandra-plugins directory)::
 
-  > load artifact target/cassandra-plugins-1.0-SNAPSHOT-batch.jar \
+  > load artifact target/cassandra-plugins-1.0-0-batch.jar \
          config-file resources/cassandra-plugin-batch.json
```
Comment (albertshau) [inline README.rst]: seems like these config filenames are wrong. the ones in the directory have the version at the end.

**Comment by sreevatsanraman** [comment]
lgtm

**Comment by bdmogal** [comment]
Squashed commits. merging.

**Pull request merged**